app/src/main/java/org/mozilla/rocket/privately/browse/BrowserFragment.kt

```diff
@@ -377,11 +379,18 @@ class BrowserFragment : LocaleAwareFragment(),
         override fun onDownloadStart(download: Download) {
             if (!TextUtils.isEmpty(download.url)) {
                 val cookie = CookieManager.getInstance().getCookie(download.getUrl())
+                val fileName: String?
```

Comment:
is fileName nullable?
maybe fileName?.let{
//  move DownloadManager.Request....code here?
}